changes made to more specific imports

tamandeeps opened the pull request:
Fixes #3602 
Changes made in this Pull Request:
 - 


PR Checklist
------------
 - [ ] Tests?
 - [x] Docs?
 - [x] CHANGELOG updated?
 - [x] Issue raised/referenced?

tamandeeps committed:
changes made to more specific imports

orbeckst commented on package/doc/sphinx/source/documentation_pages/overview.rst:
```diff
@@ -116,14 +116,14 @@ elementary and fundamental object in MDAnalysis.
 
   >>> calphas = u.select_atoms("name CA")
-  >>> print calphas
+  >>> print (calphas)
```
please revert, covered in PR #3588

tamandeeps replied:
Yes sir , changes applied.

orbeckst commented on package/doc/sphinx/source/documentation_pages/overview.rst:
```diff
@@ -116,14 +116,14 @@ elementary and fundamental object in MDAnalysis.
 over time::
 
-  >>> print u.trajectory
+  >>> print (u.trajectory)
```
please revert, covered in PR #3588

tamandeeps replied:
Done sir

orbeckst commented on package/doc/sphinx/source/documentation_pages/overview.rst:
```diff
@@ -104,7 +104,7 @@ The :class:`~MDAnalysis.core.universe.Universe` contains a number of important a
 :attr:`~MDAnalysis.core.universe.Universe.atoms`::
 
-  >>> print u.atoms
+  >>> print (u.atoms)
```
please revert, covered in PR #3588

tamandeeps replied:
Done sir.

orbeckst reviewed (changes requested):
The changes you made spot-on, well done.

I liked your initiative of fixing Python 2 syntax but because this is already fixed in PR #3588, which will be merged soon, please revert those changes and keep your PR focused on the specific issue.

Please also
- add your name to AUTHORS
- add your Github handle to the CHANGELOG, at the end of the list of contributors for 2.2.0

tamandeeps committed:
Resolved the conflict with PR #3588

orbeckst commented:
@tamandeeps I'll have a look at the changes in a bit.

There's no need to use "Sir" or any other honorifics. Everybody in the MDAnalysis Project addresses each other by first name or chosen handle. There's more explanation in the recent mailing list post [GSOC and Outreachy applicants: How to be a Good Member of a Scientific Software Community](https://groups.google.com/g/mdnalysis-devel/c/gWkydNDuc9A/m/2DMR7Dg9BwAJ) (which was posted before you joined). Please have a look.

orbeckst commented:
You completed your task, you still have to

*    add your name to AUTHORS
*    add your Github handle to the CHANGELOG, at the end of the list of contributors for 2.2.0

tamandeeps commented:
> 

Actually , I was feeling shy to add names to the AUTHORS because first I wanted to make a very significant contribution to this project as this contribution which I made now is very small.

tamandeeps committed:
Added name to Authors and github handle in Changelog

tamandeeps commented:
> > 
> 
> Yes actually , I was feeling shy to add names to the AUTHORS because first I wanted to make a very significant contribution to this project as this contribution which I made now is very small.

I have made the following changes

tamandeeps commented:
> You completed your task, you still have to
> 
> * add your name to AUTHORS
> * add your Github handle to the CHANGELOG, at the end of the list of contributors for 2.2.0

I made the following changes. Added my name to Authors and added my github handle to CHANGELOG

orbeckst commented:
Well done, congratulations on your first merged PR!

orbeckst commented:
> Actually , I was feeling shy to add names to the AUTHORS because first I wanted to make a very significant contribution to this project as this contribution which I made now is very small.

We are happy about and welcome _every_ contribution that improves MDAnalysis and one way to make this clear is to acknowledge each contributor by having their name included. We hope to see more from you in the future!!
